libs/clist/clist.go

```diff
@@ -313,7 +314,7 @@ func (l *CList) WaitChan() <-chan struct{} {
 }
 
 // Panics if list grows beyond its max length.
-func (l *CList) PushBack(v interface{}) *CElement {
+func (l *CList) PushBackWithPriority(v interface{}, priority int64) *CElement {
```

Comment:
assume priority is always positive, should be using `uint64`? can you also change the proto definition in abcix to use `uint64`?